config/privacy/privacyConfig.go

```diff
@@ -64,6 +65,15 @@ type Instagram struct {
 	Simple bool
 }
 
+// Mastodon holds the privacy configuration settings related to the Mastodon shortcode.
+type Mastodon struct {
+	Service `mapstructure:",squash"`
+
+	// TODO: revisit it
+	// If simple mode is enabled, a static and no-JS version of the Toot will be built.
```

Comment:
This is obviously not implemented yet. What is the idea here? To load `https://{{ $server}}/api/v1/statuses/{{ $tootID }}/` and statically render something based on it?

Should this be a fully styled layout or has it to be overridden by the user?

Reply:
@weddige thanks for review, I copied it from another structs, so we can potentially remove it.